app/helpers/recruitment_cycle_helper.rb

```diff
@@ -48,4 +48,8 @@ def current_cycle_provider(provider)
   def next_cycle_provider(provider)
     RecruitmentCycle.next.providers.find_by(provider_code: provider.provider_code)
   end
+
+  def tda_active?(provider_or_course)
+    provider_or_course.recruitment_cycle_year.to_i > 2024 && FeatureService.enabled?(:teacher_degree_apprenticeship)
+  end
```

Comment:
Discuss during pairing, it makes sense to move to provider and delegates from the course.